packages/sync/test/OfflineQueueLink.test.ts

```diff
@@ -98,6 +98,48 @@ describe("OnOffLink", () => {
     expect(testLink.operations.length).eqls(1);
   });
 
+  it("test online only directive skips queue", () => {
+    let operations: any[] = [];
```

Comment:
[ProTip] Normally this could land in beforeEach method as it common for all tests